### ISSUE TITLE
Reguły cenowe w /sale/offer-events

### ISSUE THREAD

**Issue by kontakt-electrobit** [allegro/allegro-api#10067]
Dzień dobry

Czy po automatycznej zmianie ceny towaru przez Reguły cenowe (FOLLOW_BY_ALLEGRO_MIN_PRICE), zmiana ta jest widoczna w /sale/offer-events jako OFFER_PRICE_CHANGED?

Dodatkowo, czy istnieje w API możliwość sprawdzenia czy dana oferta aktualnie ma zmienioną cenę przez Reguły cenowe? Chciałbym mieć możliwość odfiltrowania ofert, dla których cena nie jest automatycznie zmieniana przez brak ofert spełniających warunki (np. brak żadnej oferty z darmową dostawą)

**Comment by AureliuszBrussy** [comment]
Tak, zmian ceny w wyniku przelicznika jest widoczna w dzienniku zdarzeń. Więcej [przeczytasz w newsie](https://github.com/allegro/allegro-api/issues/9129).

Co do sprawdzenia przypisania reguł cenowych: obecnie jedyną możliwością jest skorzystanie z [GET /sale/price-automation/offers/{offerId}/rules](https://developer.allegro.pl/documentation#operation/getAutomaticPricingRulesForOfferUsingGET).

Jeżeli masz jakiś pomysł i sugestię, co moglibyśmy zrobić, aby ułatwić Tobie pracę - w zakładce [Discussions](https://github.com/allegro/allegro-api/discussions) możesz przekazać swoją propozycję. Być może na jej podstawie wdrożymy taką zmianę.

**Comment by kontakt-electrobit** [comment]
Dziękuję za odpowiedź.

Jeszcze jedno pytanie - załóżmy, że dla towaru z ustawionym FOLLOW_BY_ALLEGRO_MIN_PRICE ustawiam minimalny zakres cenowy na 100 zł. Wybrany towar nie ma automatycznie zmienionej ceny przez reguły, ze względu na brak ofert spełniających warunki. Kilka dni później muszę zwiększyć minimalny zakres na np. 120zł. Czy teraz w przypadku braku ofert do zadziałania automatycznej reguły, cena zostanie zwiększona do 120 zł, czy musiałbym ustawić ręcznie taką cenę, co z kolei wyłączy automatyczne reguły dla tej oferty?

**Comment by AureliuszBrussy** [comment]
To zależy od tego, jaka w danym momencie jest najniższa cena tego konkretnego produktu na Allegro. W tym konkretnym przypadku zakładasz, że nie ma żadnej konkurencyjnej oferty?

**Comment by kontakt-electrobit** [comment]
Tak, jak zachowa się automat, gdy chcę zwiększyć minimalną cenę w zakresie, a aktualnie nie ma innej oferty, która spełnia wymagania "Najniższa cena na Allegro" (np. żadna dostępna oferta nie ma darmowej dostawy)

**Comment by AureliuszBrussy** [comment]
Działanie reguły FOLLOW_BY_ALLEGRO_MIN_PRICE opiera się na istnieniu ceny minimalnej produktu, jeżeli taka cena nie istnieje to nie ma wykonywanych przeliczeń. Regułę czytamy “podążaj za najniższą ceną produktu w ramach zdefiniowanego zakresu” - jeżeli nie ma ceny minimalnej to nie robimy przeliczeń. W takim przypadku więc nie zmienimy ceny.
